CONTRIBUTING.md

```diff
@@ -344,9 +344,59 @@ Details for each point and good commit message examples can be found on https://
 
 ## Committers
 
-Presto committers are defined as [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and documented in the project's [`CODEOWNERS`](CODEOWNERS) file.  Each line in the `CODEOWNERS` file defines a module or submodule that the committer has the rights to approve.  New modules and submodules for CODEOWNERS may be added as needed.
+Presto has two levels of committers: module committers, and project committers.  Presto committers are defined as [code owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and documented in the project's [`CODEOWNERS`](CODEOWNERS) file, and are either directly referenced in the file, or through a Github team. Each line in the `CODEOWNERS` file defines a module or submodule that the committer or team has the rights to approve.  New modules and submodules for [`CODEOWNERS`](CODEOWNERS)  may be added as needed.
 
-New committers are approved by majority vote of the TSC ([see TSC charter](https://github.com/prestodb/tsc/blob/master/CHARTER.md)).  To become a committer, reach out to an [existing TSC member](https://github.com/prestodb/tsc#members) and ask for their feedback on your eligibility (see: [How to become a Presto Committer?](https://github.com/prestodb/presto/wiki/How-to-become-a-Presto-committer%3F)).  Note: to expedite the process, consider creating a document that outlines your Github stats, such as the number of reviews, lines of code added, number of PRs, and outlines particularly outstanding code and review contributions.  If the TSC member believes you are eligible, they will submit your nomination to a vote by the TSC, typically in the form of a PR that adds your handle to the `CODEOWNERS` file.  The process is complete once the PR is merged.
+### Baseline expectations from committers
+
+Both module and project committers must demonstrate technical mastery and commitment to the project’s community and growth.  In particular, they must show evidence that they are aligned with the goals outlined in [ARCHITECTURE](ARCHITECTURE.md), and demonstrate kindness and professionalism; help others in the project; and work collaboratively as a team.
```

Comment:
Honestly I wonder whether "growth" is a terminal goal. There's something to be said for mature projects that serve a definite purpose and are stable and maintained over years, but don't grow.

I think Presto still has plenty of room to grow, but there is such a thing as a project that is simply finished.

Reply:
I agree that growth is not a terminal goal.  I think instead, we want diversification, both in terms of the number of key contributors, and the diversity of employers of these key contributors.  We need to make sure the project survives specific individuals and specific companies.  We do have a diversity of committers compared with some competing projects, but in my opinion it's still not enough.

So, for now, I like "growth", because to achieve this diversity will require growth, but open to your suggestion on how to phrase this.

Reply:
I think we should describe the current state of the Presto project. I agree growth is not a terminal goal, and also that Presto has plenty of room to grow so I think we are a long way away from being a maintained but not-growing project. However, I also see 

"I think instead, we want diversification..." 

I have a two-part suggestion to consider: 

* Perhaps something similar to this, separating "aligned with the project’s community and growth" from "technical mastery" and grouping it with the other valued characteristics: 

  "Both module and project committers must demonstrate technical mastery of at least their core areas. They must also show evidence that they are aligned with the project’s community and growth, including the goals outlined in [ARCHITECTURE](ARCHITECTURE.md), demonstrating kindness and professionalism; helping others in the project; and working collaboratively as a team."
  
  (as an additional suggestion, "including" could be followed by a colon, and a four-item unordered list of "the goals", "demonstrating", "helping", and "working" to emphasize these topics for potential committers to find more easily)

* If what we want is diversification, let's say so, and add this explanation of diversification and "make sure the project survives specific individuals and specific companies" to ARCHITECTURE.md.

Reply:
Added that, PTAL!